src/users/interface/patch-user-interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface IPatchUser {
+  firstName?: string;
+  lastName?: string;
+  age?: number;
+  isStudent?: boolean;
+}
```

Comment:
possible to write like:
export type IUpdateUserInput = Partial<Omit<IUser, 'id'>>;

But your implementation is okay also